lib/cocoapods-core/specification/linter.rb

```diff
@@ -99,6 +100,22 @@ def validate_root_name
         end
       end
 
+      # Generates a warning if the requires_arc attribute has true or false string values.
+      #
+      # @return [void]
+      #
+      def check_requires_arc_attribute
+        attributes = DSL.attributes.values
+        attributes.each do |attr|
```

Comment:
can you just use `attributes.find { |attr| attr.name == :requires_arc }` instead?